cli/cli-client/src/main/resources/help/benchmark.adoc

```diff
@@ -0,0 +1,71 @@
+BENCHMARK(1)
+============
+:doctype: manpage
+
+
+NAME
+----
+benchmark - runs a performance benchmark against a cache.
+
+The benchmark can be performed both for HTTP and for Hot Rod: the mode is determined by the protocol of the supplied URI.
+Supported protocols are http, https, hotrod, hotrods. If unspecified it will use the current connection's URI. Hot Rod benchmarks will connect to the entire cluster. HTTP benchmarks will connect only to a single node.
+The benchmark is performed against an already-existing cache: by using a cache with the desired configuration it is possible to exercise specific features such as persistence, cross-site replication and transactions.
+
+
+SYNOPSIS
+--------
+*benchmark* ['OPTIONS'] [`uri`]
+
+
+BENCHMARK OPTIONS
+-----------------
+*-f, --forks*='num'::
+The number of forks to create. Defaults to 0 (don't use forks).
+
+*-t, --threads*='num'::
+The number of threads to create. Defaults to 10.
+
+*--cache*='cache'::
+The name of the cache to use for the benchmark. Defaults to 'benchmark'.
+
+**--key-size*='num'::
+The size of the key. Defaults to 16 bytes.
+
+**--value-size*='num'::
+The size of the value. Defaults to 1000 bytes.
+
+**--keyset-size*='num'::
+The size of the test key set. Defaults to 1000.
+
+*--verbosity*=['SILENT', 'NORMAL', 'EXTRA']::
+How verbose the output should be. One of SILENT, NORMAL, EXTRA. Defaults to NORMAL.
```

Comment:
There seems to be no difference in the output when I set different verbosity levels.